kobo/apps/accounts/migrations/0007_add_providers_from_environment_to_db.py

```diff
@@ -68,11 +77,12 @@ def add_OIDC_settings_from_env(apps, schema_editor):
 class Migration(migrations.Migration):
     dependencies = [
         ('socialaccount', '0005_socialtoken_nullable_app'),
-        ('accounts', '0005_do_nothing'),
+        ('accounts', '0006_alter_emailcontent_unique_together'),
     ]
 
     operations = [
         migrations.RunPython(
-            code=add_OIDC_settings_from_env, reverse_code=migrations.RunPython.noop
+            code=add_OIDC_settings_from_env,
+            reverse_code=migrations.RunPython.noop,
```

Comment:
We discussed placing a warning on the backwards migration as it risks losing the social app provider field.